src/main/scala/org/broadinstitute/dsde/agora/server/business/AgoraBusiness.scala

```diff
@@ -105,13 +105,18 @@ class AgoraBusiness {
         // Passed basic validation.  Now check if (any) docker images that are referenced exist
         namespace.tasks.foreach { validateDockerImage }
       case AgoraEntityType.Configuration =>
-        val json = agoraEntity.payload.get.parseJson
-        val fields = json.asJsObject.getFields("methodStoreMethod")
-        require(fields.size == 1)
-        val subFields = fields(0).asJsObject.getFields("methodNamespace", "methodName", "methodVersion")
-        require(subFields(0).isInstanceOf[JsString])
-        require(subFields(1).isInstanceOf[JsString])
-        require(subFields(2).isInstanceOf[JsNumber])
+        try {
```

Comment:
why the try/ catch here? cant you just catch the IllegalArgumentException at the supervisor level. Then you can remove the ValidationException class altogether